components/eamxx/docs/user/model_output.md

```diff
@@ -73,49 +78,57 @@ I/O interface of EAMxx. There are two types of diagnostic outputs:
   that EAMxx does not keep in persistent storage. As of May 2024, the available
   derived quantities are (case sensitive):
 
-    - `PotentialTemperature`
-    - `AtmosphereDensity`
-    - `Exner`
-    - `VirtualTemperature`
-    - `z_int`
-    - `z_mid`
-    - `geopotential_int`
-    - `geopotential_mid`
-    - `dz`
-    - `DryStaticEnergy`
-    - `SeaLevelPressure`
-    - `LiqWaterPath`
-    - `IceWaterPath`
-    - `VapWaterPath`
-    - `RainWaterPath`
-    - `RimeWaterPath`
-    - `ShortwaveCloudForcing`
-    - `LongwaveCloudForcing`
-    - `RelativeHumidity`
-    - `ZonalVapFlux`
-    - `MeridionalVapFlux`
-    - `precip_liq_surf_mass_flux`
-    - `precip_ice_surf_mass_flux`
-    - `precip_total_surf_mass_flux`
-    - `surface_upward_latent_heat_flux`
-    - `wind_speed`
-    - `AerosolOpticalDepth550nm`
-    - `NumberPath`
-    - `AeroComCld`
-
-  TODO: add some information about what each diagnostic is, perhaps a formula
+  - `Exner`: $\Pi = \left( \dfrac{p}{p_0} \right) ^\dfrac{R_d}{c_p}$
+  - `PotentialTemperature`: $\Theta = T / \Pi$
+  - `LiqPotentialTemperature`: $\Theta_\ell = \Theta - \dfrac{\Theta}{T} \dfrac{L_v}{c_p}q_c$
+  - `dz`: $dz = \dfrac{rd}{g}\dfrac{\rho_s T_v}{p}$
+  - `z_int`: vertical sum of `dz` from surface (b.c. at surface: 0 or $\phi_s/g$, depending if from surface or sealevel)
+  - `z_mid`: midpoint average of `z_int`
+  - `geopotential_int`: vertical sum of `g dz` from surface (b.c. at surface: $\phi_s$)
+  - `geopotential_mid`: midpoint average of `geopotential_int`
+  - `AtmosphereDensity`: $\rho = \dfrac{\rho_s}{dz*g}
+  - `VirtualTemperature`: $T_v = T\left(1+c_1 q_v\right)$
+  - `DryStaticEnergy`: $DSE = c_p T + gz + \phi_s$
+  - `SeaLevelPressure`
```

Comment:
The explanation of SeaLevelPressure is insanely long. There is a whole tex file buried in our docs folder explaining the formula.

I am having second thoughts about adding formulas for all the diagnostics. Should I continue (skipping p_sl and any other with too long formulas)? Or stop, and remove all these formulas?

Reply:
The standardname should now be part of the variables metadata. But yeah, one needs to know what the heck is the eamxx name for `atmosphere_mass_content_of_water_vapor` in order to even start requesting it in an output stream...